smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -826,6 +829,28 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
               API.SCARD_ATTR_ATR_STRING)
           .$waitAndVerify();
     },
+
+    // Test that SetAttrib fails for unsupported attributes.
+    'testSetAttrib_error': async function() {
+      expectReportPlainResult(/*requestId=*/ 111, 'INVALID_PARAMETER');
+
+      await mockChromeApi
+          .dispatchEvent(
+              'onSetAttribRequested', /*requestId=*/ 111, readerHandle,
+              API.SCARD_ATTR_ATR_STRING, /*data=*/ new ArrayBuffer(0))
+          .$waitAndVerify();
+    },
+
+    // Test that SetAttrib works for supported attribute.
+    'testSetAttrib_success': async function() {
+      expectReportPlainResult(/*requestId=*/ 111, 'SUCCESS');
+
+      await mockChromeApi
+          .dispatchEvent(
+              'onSetAttribRequested', /*requestId=*/ 111, readerHandle,
+              TAG_IFD_DEVICE_REMOVED, /*data=*/ new Uint8Array(['0x0']))
```

Comment:
(Looks like the byte has to be nonzero to actually cause something to happen in the driver, but I guess it's not the point of the test here. It's enough that we verify the "success" here. If you can maybe drop a comment about this for future readers, but I don't mind if you land without it.)